tests/app/celery/test_ftp_update_tasks.py

```diff
@@ -91,6 +95,20 @@ def test_update_letter_notifications_statuses_builds_updates_list(notify_api, mo
     assert updates[1].cost_threshold == 'Sorted'
 
 
+def test_update_letter_notifications_statuses_persisted(notify_api, mocker, sample_letter_template):
+    sent_letter = create_notification(sample_letter_template, reference='ref-foo', status=NOTIFICATION_SENDING)
+    failed_letter = create_notification(sample_letter_template, reference='ref-bar', status=NOTIFICATION_SENDING)
+
+    valid_file = '{}|Sent|1|Unsorted\n{}|Failed|2|Sorted'.format(
```

Comment:
asset they are both in sending first?

Reply:
Not sure if that proves anything, as they are both explicitly set to sending?